pkg/identity/certificate_authority.go

```diff
@@ -53,7 +53,7 @@ type CASetupConfig struct {
 	KeyPath        string `help:"path to the private key for this identity" default:"$IDENTITYDIR/ca.key"`
 	Difficulty     uint64 `help:"minimum difficulty for identity generation" default:"30"`
 	Timeout        string `help:"timeout for CA generation; golang duration string (0 no timeout)" default:"5m"`
-	Overwrite      bool   `help:"if true, existing CA certs AND keys will overwritten" default:"false"`
+	Overwrite      bool   `help:"if true, existing CA certs AND keys will overwritten" default:"false" setup:"true"`
```

Comment:
Do we still want to support overwrite here?

Reply:
^